stubs/Pillow/PIL/Image.pyi

```diff
@@ -224,18 +234,18 @@ def register_extensions(id: str, extensions: Iterable[str]) -> None: ...
 def registered_extensions() -> dict[str, str]: ...
 def register_decoder(name: str, decoder) -> None: ...
 def register_encoder(name: str, encoder) -> None: ...
-def effect_mandelbrot(size: tuple[int, int], extent, quality) -> Image: ...
-def effect_noise(size: tuple[int, int], sigma) -> Image: ...
-def linear_gradient(mode) -> Image: ...
-def radial_gradient(mode) -> Image: ...
+def effect_mandelbrot(size: tuple[int, int], extent: tuple[float, float, float, float], quality: int) -> Image: ...
+def effect_noise(size: tuple[int, int], sigma: float) -> Image: ...
+def linear_gradient(mode: str) -> Image: ...
+def radial_gradient(mode: str) -> Image: ...
 
 class Exif(MutableMapping[int, Any]):
     def load(self, data: bytes) -> None: ...
     def tobytes(self, offset: int = ...) -> bytes: ...
-    def get_ifd(self, tag: int): ...
+    def get_ifd(self, tag: int) -> Any: ...
```

Comment:
I'd prefer to leave this as unannotated if we haven't figured out the precise type yet. It seems like it returns a dict of some sort.